static/js/tools/map/util.ts

```diff
@@ -122,6 +128,13 @@ export const ENCLOSED_PLACE_TYPE_NAMES = {
 export const USA_PLACE_HIERARCHY = ["Country", "State", "County"];
 export const MAP_REDIRECT_PREFIX = "/tools/map";
 
+export interface DataPointMetadata {
```

Comment:
please add brief comments for these, especially for shared interfaces

Reply:
done